pkg/app.go

```diff
@@ -239,6 +261,187 @@ func (app *App) Run(mws ...MiddleWare) {
 	<-endRunning
 }
 
+// Run beego services individually.
+func (app *App) RunServices(mws ...MiddleWare) {
```

Comment:
> From this method, it seems that we cannot listen to different ports.

Yes this is the method to run all services registered to the `App` in one go, so I'm not sure if it could take ports as an argument. Since with this PR we can only run one service for now, (we fetch the port in the default way from `BConfig`).
I was hoping to work on the config and different ports issue in the next PR for this issue since I'm still wrapping around my head on how configuration works in beego.